lib/components/Columns/Columns.treat.ts

```diff
@@ -11,6 +11,18 @@ export const gutterOffset = styleMap(theme =>
   ),
 );
 
+export const gutterOffsetDesktop = styleMap(theme => ({
+  none: theme.utils.desktopStyles({ marginLeft: 0 }),
+  ...mapToStyleProperty(
+    theme.space,
+    'marginLeft',
+    (space: number, propertyName) =>
```

Comment:
`space` can be implicitly typed here. Feels weird mixing both concepts in one function.